tests/core/conversions/test_dataframe.py

```diff
@@ -64,7 +65,7 @@ def ndf():
 
 
 class TestConversionToDataFrame:
-
+    
```

Comment:
trailing spaces are checked by the linting, the best is to also run them locally before pushing:
```
pycodestyle kgforge
pylint -j2 kgforge --fail-under=9.0
```